liquibase-standard/src/main/java/liquibase/command/core/DropAllCommandStep.java

```diff
@@ -1,49 +1,37 @@
 package liquibase.command.core;
 
+import liquibase.CatalogAndSchema;
+import liquibase.Scope;
 import liquibase.command.*;
-import liquibase.configuration.ConfigurationValueObfuscator;
-import liquibase.exception.CommandExecutionException;
+import liquibase.database.Database;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.LiquibaseException;
+import liquibase.lockservice.LockService;
+import liquibase.logging.Logger;
+import liquibase.util.StringUtil;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.UUID;
 
-public class DropAllCommandStep extends AbstractCliWrapperCommandStep {
+public class DropAllCommandStep extends AbstractCommandStep {
 
     public static final String[] COMMAND_NAME = {"dropAll"};
 
-    public static final CommandArgumentDefinition<String> USERNAME_ARG;
-    public static final CommandArgumentDefinition<String> PASSWORD_ARG;
-    public static final CommandArgumentDefinition<String> URL_ARG;
-    public static final CommandArgumentDefinition<String> DEFAULT_CATALOG_NAME_ARG;
-    public static final CommandArgumentDefinition<String> DEFAULT_SCHEMA_NAME_ARG;
+    private static final Logger LOG = Scope.getCurrentScope().getLog(DropAllCommandStep.class);
```

Comment:
I want to say that getting the log as a class level static variable might be problematic for structured logging, because it might execute before we've had a chance to set up structured logging. I am not positive of this. Was this just a convenience change?

Reply:
That's good to know, let me fix it then. But there is sonar minor warning for this, 
so I it was force of habit -> https://rules.sonarsource.com/java/RSPEC-1312 . In web project it's common use to declare it as static too.

Reply:
Well... I am not positive of this. Maybe it's worth experimenting with at some point.

Reply:
No problems, and thanks for noticing that.